CONTRIBUTING.rst

```diff
@@ -11,10 +11,7 @@ http://docs.stackstorm.com/development/index.html
 Managing Python dependencies
 ----------------------------
 
-.. note::
-
-    ``requirements.txt`` files are generated automatically using
-    ``scripts/fixate-requirements.py`` script and should't be edited manually.
+    ``requirements.txt`` files are generated automatically using ``scripts/fixate-requirements.py`` script and should't be edited manually.
```

Comment:
you gonna fix the other typo in there while you're at it? /s/should't/shouldn't/.